Code/GraphMol/Fingerprints/MorganFingerprints.cpp

```diff
@@ -315,6 +316,9 @@ SparseIntVect<uint32_t> *getHashedFingerprint(
     std::vector<uint32_t> *invariants, const std::vector<uint32_t> *fromAtoms,
     bool useChirality, bool useBondTypes, bool onlyNonzeroInvariants,
     BitInfoMap *atomsSettingBits, bool includeRedundantEnvironments) {
+  if(nBits == 0) {
```

Comment:
You've added a test for this in the Python wrapper (which is great, thanks!), but it would also not be bad to add one in the C++ tests.
This isn't necessary, but if you want to do it, the logical place would be to add a new `TEST_CASE` to `Code/Fingerprints/catch_tests.cpp`

Reply:
Hi Greg, I created a new C++ test case. Can you have a look at it?